### DISCUSSION TITLE
State Attorney General cases

### DISCUSSION THREAD

**Discussion by milanphdworkskhop** [freelawproject/courtlistener#3402] [Q&A]
How could I retrieve systematically all state AG settlements and enforcement actions? Political scientist Paul Nolette built this database manually ([https://attorneysgeneral.org/settlements-and-enforcement-actions/](https://attorneysgeneral.org/settlements-and-enforcement-actions/)). I would like to re-construct his data, but with Court Listener data. How could I do it

**Comment by flooie** [comment]
@flowerpower13 - unfortunately that isnt something you can do with courtlistener.  We do collect a handful of AG opinions from a few states, but I'm not sure that is what you are looking for.  

A few months ago I created a juriscraper branch [Add AG to Juriscraper](https://github.com/freelawproject/juriscraper/pull/647) to facilitate the collection of all available state attorney general opinions, but the PR has languished in need of a volunteer to review it.  I believe a number of the scrapers no longer work as websites have changed since then.  

Since we dont have the data you cant do it here, but if you are committed you could do something similar to the juriscraper branch if this data is available.
